lib/dust-helpers.js

```diff
@@ -270,42 +270,45 @@ var helpers = {
               return null;
           };
 
-      key  = dust.helpers.tap(key, chunk, context);
-      operand = dust.helpers.tap(operand, chunk, context);
+      key = parseFloat(dust.helpers.tap(key, chunk, context));
+      operand = parseFloat(dust.helpers.tap(operand, chunk, context));
```

Comment:
Shouldn't need to. Anything icky coerces to `NaN` and that's the eventual math output.